packages/react-instantsearch-dom/src/widgets/__tests__/Answers.js

```diff
@@ -42,6 +42,7 @@ describe('Answers', () => {
     `);
   });
 
+  // eslint-disable-next-line jest/expect-expect
```

Comment:
I'm fine with not tackling this right now, but let's add a `@TODO` so we know it should be handled at some point. The test looks incomplete, or at least not explicit.

Reply:
We will deprecate/remove Answers soon so I'm not sure it's worth updating this test.

Reply:
Makes sense 👍